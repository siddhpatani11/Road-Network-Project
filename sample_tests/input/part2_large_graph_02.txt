114 343
0 42 15 G
0 58 42 R
1 19 22 G
1 21 35 B
1 22 43 R
1 85 1 G
1 108 3 B
2 7 2 B
2 70 40 G
2 78 43 B
3 98 6 R
4 21 28 G
4 74 26 B
4 106 36 G
5 44 22 R
5 111 21 B
6 0 7 B
6 26 3 R
6 70 43 R
6 81 50 B
6 113 11 R
7 22 34 B
7 88 15 B
7 89 47 G
7 97 22 G
8 31 24 G
8 88 18 B
9 35 45 B
9 49 42 R
9 74 25 G
10 12 8 G
10 36 16 G
10 86 9 B
11 31 48 R
11 42 9 G
11 44 10 G
12 16 22 G
12 41 2 G
12 72 37 G
13 11 45 G
13 51 8 G
13 67 10 G
13 81 23 B
14 15 42 B
14 25 34 B
14 54 24 B
15 74 15 B
16 23 24 R
17 91 29 B
17 101 43 G
19 28 22 B
19 49 1 G
19 96 43 G
20 39 32 B
21 16 33 R
21 65 22 B
22 41 37 R
22 47 49 G
22 80 6 R
23 12 35 B
23 40 11 B
23 110 19 G
24 58 28 G
24 73 40 R
25 63 36 G
26 39 47 R
26 55 11 G
26 94 45 G
27 54 44 B
28 76 16 G
28 102 48 R
29 57 24 B
29 76 35 R
30 12 34 B
30 52 47 B
30 82 38 B
31 1 15 R
31 72 8 R
31 88 2 B
31 89 49 B
31 95 41 R
32 0 24 G
32 12 5 B
32 69 24 B
33 31 3 B
33 94 15 G
34 22 34 B
34 64 48 B
35 12 28 B
35 98 37 G
36 88 38 B
36 92 44 B
37 4 18 B
37 26 5 G
37 33 1 R
37 73 42 G
37 111 42 B
38 31 11 R
38 36 4 R
39 57 3 G
39 62 5 G
39 67 25 B
39 85 6 G
39 87 24 B
40 11 22 B
40 20 27 G
40 74 7 B
40 85 12 R
40 105 34 R
40 110 39 G
41 28 47 B
41 48 29 G
41 52 46 R
41 68 22 G
41 88 33 B
42 33 4 G
42 44 7 B
42 55 11 R
42 59 27 R
42 104 51 B
43 11 35 G
43 68 49 R
43 104 29 G
44 47 32 G
44 72 16 B
44 78 29 B
44 89 48 G
44 107 3 B
45 8 40 B
45 22 37 B
45 75 50 G
45 101 42 B
45 111 49 R
46 11 31 G
46 63 6 B
46 84 19 R
46 103 10 B
47 12 6 R
47 46 7 B
47 86 6 G
47 104 45 G
48 15 12 B
48 24 18 G
48 86 38 B
48 113 36 B
49 0 6 B
50 20 18 R
50 95 29 R
51 33 8 G
51 34 24 G
51 48 13 B
51 96 36 R
51 100 49 G
52 37 41 G
52 42 17 R
52 65 49 R
52 78 39 R
52 108 31 R
53 7 11 G
53 89 32 R
54 25 47 B
54 34 10 R
54 81 10 R
54 90 15 B
55 27 3 B
55 46 9 B
55 71 30 B
56 53 23 B
56 63 41 B
56 95 45 G
57 6 40 G
57 56 34 B
57 83 17 G
58 32 6 B
58 60 10 G
58 69 6 R
58 102 27 R
58 106 13 G
59 4 18 B
59 80 47 G
60 2 39 G
60 51 30 B
60 56 7 R
61 42 37 R
62 41 32 R
62 52 18 R
62 73 7 B
62 77 29 R
63 16 11 R
63 20 18 G
63 21 46 G
63 24 17 R
63 31 8 B
63 35 15 R
63 45 13 G
63 60 49 B
63 88 42 G
64 55 43 B
64 94 18 R
65 16 45 R
65 28 22 B
65 41 19 G
65 70 15 B
65 71 38 G
66 13 29 R
66 27 19 R
66 61 17 R
66 64 9 B
67 64 42 B
67 105 21 G
67 110 17 R
67 112 43 B
68 16 15 B
69 23 10 B
69 35 19 B
69 107 14 R
70 23 12 R
71 30 19 R
71 81 3 R
72 9 25 B
72 24 50 B
72 87 38 R
73 86 19 B
74 14 6 R
74 67 22 B
74 83 25 G
74 108 1 B
75 15 7 B
75 28 2 G
75 30 13 G
75 58 17 G
76 100 30 R
77 37 6 B
77 90 18 R
78 0 2 G
78 18 15 B
78 29 35 R
78 39 49 R
78 41 7 G
78 69 28 B
78 87 36 B
78 88 35 R
79 61 17 R
79 85 12 R
80 28 43 G
81 12 42 R
81 107 32 G
82 11 41 R
82 17 44 R
82 61 11 B
82 74 15 G
82 86 24 B
83 4 28 G
83 17 41 G
83 55 41 R
84 51 20 R
85 17 10 R
86 1 9 G
86 31 9 R
86 63 39 R
86 70 40 R
86 113 39 R
87 20 44 B
88 8 42 G
88 25 3 G
88 94 23 G
88 102 34 B
89 63 27 B
89 106 26 B
90 3 9 B
91 78 10 B
91 93 49 R
92 83 40 R
92 113 39 R
93 64 50 B
94 21 42 G
94 36 45 R
94 45 36 R
94 91 16 G
94 107 46 R
95 21 10 R
95 40 34 G
95 78 18 R
96 6 21 R
96 18 8 B
96 57 28 R
96 58 36 G
96 68 31 R
96 76 6 B
96 95 23 R
98 9 1 G
98 82 27 B
98 94 12 G
98 99 33 R
98 111 6 G
99 31 2 B
99 51 17 B
100 56 44 B
100 80 5 B
100 89 18 B
101 12 38 G
101 98 37 G
102 56 8 G
102 57 4 B
102 58 32 G
103 25 13 G
103 45 30 B
103 69 7 G
103 72 7 B
103 98 7 B
104 33 7 B
104 68 8 G
104 77 11 B
104 86 46 R
104 96 14 B
105 19 10 R
105 49 2 B
105 54 1 R
106 6 5 R
106 7 51 R
106 74 17 G
106 80 14 R
106 109 37 R
107 42 14 R
107 97 47 R
107 100 34 G
108 1 47 G
109 5 28 R
109 74 40 B
109 75 15 G
109 97 42 R
110 28 3 G
110 95 33 B
110 107 34 B
111 68 14 R
111 84 26 R
111 88 13 G
111 91 46 R
112 3 42 B
112 8 44 R
112 97 12 G
113 35 31 R
113 61 2 G
