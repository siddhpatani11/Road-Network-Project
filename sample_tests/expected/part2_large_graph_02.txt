16 R 96 R 113 - -1 - -1 - -1 - -1 - -1 - -1 R 481 - -1 - -1 R 195 - -1 - -1 R 186 - -1 - 0 R 368 - -1 R 135 - -1 R 86 - -1 R 24 R 262 R 442 - -1 - -1 R 148 R 439 - -1 R 98 R 234 R 95 - -1 R 260 - -1 - -1 - -1 R 453 R 173 R 101 R 91 - -1 R 375 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 R 227 - -1 - -1 R 204 R 456 R 228 - -1 - -1 - -1 R 458 R 245 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 R 106 R 465 R 180 - -1 - -1 - -1 R 94 - -1 - -1 - -1 R 324 - -1 - -1 R 114 R 471 R 429 R 134 R 259 - -1 - -1 - -1 - -1 R 462 R 76 - -1 R 124 R 297 - -1 - -1 R 411 R 196 - -1 R 425 - -1 - -1 R 77 R 483 - -1 R 43 - -1 - -1 - -1 
